### ISSUE TITLE
bug: Capacitor Cookie initial value is undefined

### ISSUE THREAD

**Issue by kjr-lh** [ionic-team/capacitor#6350]
## Bug Report

When using the capacitor cookie plugin, the initial `document.cookie` value is undefined. In browsers the initial value is an empty string. This causes some libraries (e.g. universal cookie) to assume that cookies are not available and not initialise correctly.

### Capacitor Version

```
💊   Capacitor Doctor  💊

Latest Dependencies:

  @capacitor/cli: 4.7.0
  @capacitor/core: 4.7.0
  @capacitor/android: 4.7.0
  @capacitor/ios: 4.7.0

Installed Dependencies:

  @capacitor/cli: 4.1.0
  @capacitor/core: 4.3.0
  @capacitor/ios: 4.3.0
  @capacitor/android: 4.3.0
```

### Platform(s)

android

### Other Technical Details

`npm --version` output:

`node --version` output:

`pod --version` output (iOS issues only):

**Comment by markemer** [comment]
Can you attach a small sample showing the exact issue you're having?

**Comment by bsalesc** [comment]
Potentially this got fixed here? https://github.com/ionic-team/capacitor/commit/57f8b39d7f4c5ee0e5e5cb316913e9450a81d22b

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by markemer** [comment]
> Potentially this got fixed here? [57f8b39#diff-e83e9a242653642b479198a24b977d24da86c89865825a05f8943b2ff1ccd396R78](https://github.com/ionic-team/capacitor/commit/57f8b39d7f4c5ee0e5e5cb316913e9450a81d22b#diff-e83e9a242653642b479198a24b977d24da86c89865825a05f8943b2ff1ccd396R78)

I think it might have - I'll have to see based on the sample - unless @kjr-lh wants to try themselves.

**Comment by kjr-lh** [comment]
It's fixed :-)
